docs/administration.md

```diff
@@ -227,19 +227,26 @@ is not a TTY" errors. For example:
 `docker-compose exec -T webserver document_exporter ../export`
 
 ```
-document_exporter target [-c] [-f] [-d]
+document_exporter target [-sm] [-c] [-f] [-p] [-d] [-na] [-nt]
 
 optional arguments:
+-sm, --split-manifest
 -c, --compare-checksums
 -f, --use-filename-format
+-p, --use-filename-prefix
```

Comment:
I feel like this could be better named.  It separates the various files into directories, which is more than just a change to a filename.

Reply:
Perhaps `--use-folder-prefix` since it's about folders more than filenames?

Reply:
Agreed, fixed by 7eb784d08e9bdf84f1ab6b9b41299dee23b84f80